newIDE/app/src/EventsSheet/EventsTree/Renderers/ForEachStructureEvent.js

```diff
@@ -0,0 +1,250 @@
+// @flow
+import * as React from 'react';
+import InstructionsList from '../InstructionsList.js';
+import classNames from 'classnames';
+import {
+  selectableArea,
+  executableEventContainer,
+  disabledText,
+  instructionParameter,
+  nameAndIconContainer,
+} from '../ClassNames';
+import InlinePopover from '../../InlinePopover';
+import SceneVariableField from '../../ParameterFields/SceneVariableField';
+import { type EventRendererProps } from './EventRenderer';
+import ConditionsActionsColumns from '../ConditionsActionsColumns';
+import { shouldActivate } from '../../../UI/KeyboardShortcuts/InteractionKeys.js';
+const gd: libGDevelop = global.gd;
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  instructionsContainer: {
+    display: 'flex',
+  },
+  actionsList: {
+    flex: 1,
+  },
+};
+
+export default class ForEachStructureEvent extends React.Component<
+  EventRendererProps,
+  *
+> {
+  _objectField: ?ObjectField = null;
+  state = {
+    editingVariable: false,
+    editingStructure: false,
```

Comment:
Like the renaming in the class, let's rename these 
```suggestion
    editingChildVariableName: false,
    editingIterableVariableName: false,
```